tests/UI/pages/BO/catalog/products/add.js

```diff
@@ -95,6 +96,36 @@ class AddProduct extends BOBasePage {
     await page.keyboard.type(value);
   }
 
+  /**
+   * Get Number of images to set on the product
+   * @param page {Page} Browser tab
+   * @returns {Promise<number>}
+   */
+  async getNumberOfImages(page) {
+    return (await page.$$(this.imagePreviewBlock)).length;
```

Comment:
Double dollar ?

Reply:
Yes, to get all elements matching the selector. Ref [here](https://playwright.dev/docs/api/class-page?_highlight=%24%24#pageselector-1)